libspawner/inc/system_pipe.h

```diff
@@ -3,7 +3,6 @@
 
 #include <memory>
 
-#include "error.h"

```

Comment:
Please read corresponding commit message:
> This prevents its global inclusion into unrelated sources.

It's not needed directly in the header, so I've decided to reduce scope of it's inclusion.